### ISSUE TITLE
Add syndication feed for latest additions?

### ISSUE THREAD

**Issue by cdchapman** [pyvandenbussche/lov#57]

**Comment by gatemezing** [comment]
@cdchapman  Do you think the current version of LOV implement this need?
![lov-latestupdate](https://cloud.githubusercontent.com/assets/922883/16080830/f2bb2886-330a-11e6-9f4c-3c99e6a34840.png)

**Comment by cdchapman** [comment]
@gatemezing, no, since I would like either an ATOM feed or something machine readable that I could subscribe to.
